.github/workflows/welcome-newcomers.yaml

```
@@ -0,0 +1,40 @@
+name: first-interaction
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    branches: [main]
+    types: [opened]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Welcome to the Pantsbuild Community. This looks like your first issue here.
+
+            If you haven't already, come say hi on Slack.
```

I'd personally soften this: people shouldn't feel like this is an expectation or requirement:
```suggestion
            If you haven't already, feel free to come say hi on Slack.
```